gamecenter/api/views.py

```diff
@@ -108,7 +108,7 @@ def get_paginated_scores(args):
     return scores_from_query(
         result_set=res_set.offset(args['offset'] - 1).limit(args['page_size']),
         args=args,
-        count=res_set.count(),
+        count=res_set.count()
```

Comment:
idrc, but the reason I do this is so that it's easier to add new arguments, and in the git log, it shows only the new argument in the diff, instead of also the old one to add a comma.